sdk/identity/tests.yml

```diff
@@ -8,6 +8,8 @@ extends:
         displayName: Set OIDC variables
         env:
           ARM_OIDC_TOKEN: $(ARM_OIDC_TOKEN)
+          ARM_CLIENT_ID: $(ARM_CLIENT_ID)
+          ARM_TENANT_ID: $(ARM_TENANT_ID)
```

Comment:
Why do we have to specify this twice? First here, and then later in the `EnvVars:` property group?

Reply:
I think it makes them available at different scopes - to the pipeline and to the test runner. But I'm not 100% sure actually. @benbp might have a better answer.

Reply:
@ahsonkhan the env variables are only made available within the `AzureCLI@2` task by devops (invocation of the task is what causes devops to populate the federated credential info/token). So we have to grab them first from the task, then persist them into the devops variable context, and finally we can pass them down as EnvVar parameters for other tasks to use.